{"uuid": "314d759f-2651-4455-8fb5-fcfb978d4077", "children": ["b32911e1-e2c6-48f0-9bab-97a8babddc87"], "befores": [{"name": "_xunit_setup_method_fixture_TestUserAuth", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'cookies'\n", "trace": "  File \"/Users/akre-t/Library/Python/3.9/lib/python/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/akre-t/Library/Python/3.9/lib/python/site-packages/_pytest/fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/akre-t/Library/Python/3.9/lib/python/site-packages/_pytest/fixtures.py\", line 901, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/akre-t/Library/Python/3.9/lib/python/site-packages/_pytest/python.py\", line 907, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"/Users/akre-t/Library/Python/3.9/lib/python/site-packages/_pytest/python.py\", line 788, in _call_with_optional_argument\n    func()\n  File \"/Users/akre-t/Desktop/Ренат/Coding/testQa/testQA/tests/test_auth.py\", line 25, in setup\n    self.auth_sid = self.get_cookie(response1,\"auth_sid\")\n  File \"/Users/akre-t/Desktop/Ренат/Coding/testQa/testQA/lib/base_case.py\", line 6, in get_cookie\n    assert cookie_name in response.cookies, f\"Куки был не найден с именем {cookie_name} в последнем ответе\"\n"}, "steps": [{"name": "POST request to URL '/user/login'", "status": "passed", "start": 1682237411385, "stop": 1682237411561}], "start": 1682237411385, "stop": 1682237411561}], "start": 1682237411385, "stop": 1682237411567}